src/brogue/Combat.c

```diff
@@ -510,30 +510,35 @@ boolean forceWeaponHit(creature *defender, item *theItem) {
             otherMonster = NULL;
             strcpy(buf2, tileCatalog[pmap[defender->xLoc + newLoc[0] - oldLoc[0]][defender->yLoc + newLoc[1] - oldLoc[1]].layers[highestPriorityLayer(defender->xLoc + newLoc[0] - oldLoc[0], defender->yLoc + newLoc[1] - oldLoc[1], true)]].description);
         }
+        
+        // Only deal force damage if target collides with solid object
+        forceDamage = 0;
```

Comment:
Can this be initialised to 0 when declared?

Reply:
The way the codebase seems to handle local variables is to put all the declarations at the top of the function body and wait until later to intialize. I put the `forceDamage = 0;` where I did in order to conform to this style.